app/components/Puzzle.tsx

```diff
@@ -1005,6 +1015,16 @@ export const Puzzle = ({
                   text={t`Download .puz File`}
                 />
               )}
+              <TopBarDropDownLink
+                icon={<FaMoon />}
+                text={t`Toggle Light/Dark Mode`}
+                onClick={() => toggleColorPref()}
+              />
+              <TopBarDropDownLink
+                icon={<FaMoon />}
+                text={t`Use browser/OS setting for Color Theme`}
+                onClick={() => setColorPref(null)}
+              />
```

Comment:
I've opted here to add another menu item for using the Browser/OS setting since that seems useful. But that does seem a bit redundant. Maybe it would feel less-so if that menu item used a different icon?